### ISSUE TITLE
KL loss calculation

### ISSUE THREAD

**Issue by kaleidoscopical** [yifan123/flow_grpo#78]
Thanks for making your great work open to everyone! 
I found the calculation of `kl_loss` in your code is slightly different to the KL term in your paper, with the `∆ t` missing.
Will it have an influence on the training?

`kl_loss = ((prev_sample_mean - prev_sample_mean_ref) ** 2).mean(dim=(1,2,3), keepdim=True) / (2 * std_dev_t ** 2)`
https://github.com/yifan123/flow_grpo/blob/main/scripts/train_sd3.py#L901

**Comment by yifan123** [comment]
Please refer to this [issue](https://github.com/yifan123/flow_grpo/issues/22#issuecomment-3022732047)

**Comment by kaleidoscopical** [comment]
Thanks! Very impressive
